src/scheduler/MpiWorld.cpp

```diff
@@ -1554,12 +1568,10 @@ std::shared_ptr<InMemoryMpiQueue> MpiWorld::getLocalQueue(int sendRank,
 // Note - the queues themselves perform concurrency control
 void MpiWorld::initLocalQueues()
 {
-    // Assert we only allocate queues once
-    assert(localQueues.size() == 0);
```

Comment:
I remove this assertion as now this method is also called at the end of `prepareMigration` to update the `localQueues` with the new ranks to hosts mapping.